system.asd

```diff
@@ -1,23 +1,25 @@
 ;;;; SPDX-FileCopyrightText: (#| TMPL_VAR author |#)
 ;;;; SPDX-License-Identifier: (#| TMPL_VAR license |#)
 
-(asdf:defsystem #:(#| TMPL_VAR name |#)
+(defsystem "(#| TMPL_VAR name |#)"
   :description "Describe (#| TMPL_VAR name |#) here"
   :author "(#| TMPL_VAR author |#)"
+  :homepage "https://github.com/atlas-engineer/(#| TMPL_VAR name |#)"
   :license  "(#| TMPL_VAR license |#)"
-  :version "0.0.1"
-  :serial t(#| TMPL_IF depends-on |#)
+  :version "0.0.0"(#| TMPL_IF depends-on |#)
```

Comment:
But wait, this TMPL_IF is misplaced then -- it should come right before the line with `:depends-on`, while right now it will expand the `:in-order-to` line instead. Move it to one line below?

Reply:
oops!  Fixed.